mne/io/ctf/ctf.py

```diff
@@ -57,6 +57,10 @@ def read_raw_ctf(directory, system_clock='truncate', preload=False,
     Notes
     -----
     .. versionadded:: 0.11
+    .. note:: To read in the Polhemus digitization data (for example, from
+              a .pos file), include the file in the CTF directory. The
+              points will then automatically be read into the `mne.io.Raw`
+              instace via `mne.io.read_raw_ctf`.
```

Comment:
```suggestion
              instance via `mne.io.read_raw_ctf`.
```

but now I am also confused as

https://mne.tools/stable/generated/mne.channels.read_polhemus_fastscan.html
https://mne.tools/stable/generated/mne.channels.read_dig_polhemus_isotrak.html#mne.channels.read_dig_polhemus_isotrak

do not specify .pos files.

I think we replicated the behavior of brainstorm

Reply:
Yes there is still some weirdness with how we handle these files. That's for another PR, though...